frontend/src/features/SidebarSuite/TabContent/UtilityOptionsSection/utils.ts

```diff
@@ -53,37 +32,38 @@ interface UtilityClickHandlerProps {
   };
 }
 
-type UtilityOptionProps = {
+export type UtilityOptionProps = {
   callback: (props: UtilityClickHandlerProps) => void;
   icon: OverridableComponent<SvgIconTypeMap>;
 };
 
-export type UtilityOptions = {
-  [value in UtilityOption]: UtilityOptionProps;
-};
-
 export const defaultAnchorEls = {
-  download: null,
+  download_data: null,
   copyQueryTitle: null,
   copyQueryLink: null,
   emailQueryLink: null,
 };
 
-export const onDownload = async ({
+// TODO: awaiting spec clarity
```

Comment:
for later?

Reply:
Downloads handling was only came online after. There are existing tickets for this, but I've also added it to #335.